bucket_map/src/bucket_storage.rs

```diff
@@ -427,23 +426,21 @@ impl<O: BucketOccupied> BucketStorage<O> {
                 .fetch_add(measure_flush.end_as_us(), Ordering::Relaxed);
         }
         let mut measure_mmap = Measure::start("measure_mmap");
-        let res = unsafe { MmapMut::map_mut(&data) };
-        if let Err(e) = res {
+        let mmap = unsafe { MmapMut::map_mut(&data) }.unwrap_or_else(|err| {
```

Comment:
this adds a lot more code inside `unsafe`. I'm sure you're aware. It is all error handling code.